nn/math_.py

```diff
@@ -27,6 +27,13 @@ def gelu(x: LayerRef) -> Layer:
   return activation(x, activation="gelu")
 
 
+def glu(v: LayerRef) -> Layer:
```

Comment:
I think we should also have `axis` here (with `axis="F"` as default).

Reply:
btw i am not sure if you checked issue #66. I am getting error when using `nn.split` here.